### PR TITLE
[Documentation] Database infrastructure first draft

### PR DESCRIPTION
**Pull request by cmadjar**
this is a first draft of section 4 about the backend directory structure and the database infrastructure.

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (MounaSafiHarab) [inline .idea/vcs.xml]: I wanted to ask at the KT meeting about the 2 .idea files and time was up. I will add it to the next meeting!

Comment (cmadjar) [reply .idea/vcs.xml]: I don't know about those. I based my PR out of David's PR

**Comment by cmadjar** [comment]
Thanks @davidblader! Is this better and more clear now?

File: docs/04-TechnicalInfrastructure.md [hunk 2]
```diff
@@ -1 +1,323 @@
 # 4.0 - Technical Infrastructure
+
+
+## 4.1 Back end directory structure
+
+The root directory of the imaging part of a LORIS instance is typically 
+  `/data/project`.
+
+```
+## Imaging pipeline file directory structure
```
Comment (davidblader) [inline docs/04-TechnicalInfrastructure.md]: this is much better :smiley_cat:

**Comment by cmadjar** [comment]
Closing this PR to create another one based from the proper branch.

**Pull request closed without merging**